config/foreman-proxy-content.migrations/161216113043-move_puppet_to_top_level.rb

```diff
@@ -0,0 +1,18 @@
+answers[:puppet] ||= {
+                       :server => true,
+                       :server_environments_owner => 'apache',
+                       :server_foreman_ssl_cert => '/etc/foreman/client_cert.pem',
+                       :server_foreman_ssl_key => '/etc/foreman/client_key.pem',
+                       :server_foreman_ssl_ca => '/etc/foreman/proxy_ca.pem'
+                     }
+
+answers[:puppet][:server_foreman_url] = "https://#{answers[:foreman_proxy_content][:parent_fqdn]}"
+
+scenario[:order] = [
+  'certs',
+  'foreman_proxy',
+  'foreman_proxy::plugin::pulp',
+  'foreman_proxy_content',
+  'certs::puppet',
+  'puppet'
+]
```

Comment:
If anything else changes the order, this would over-write it. Might be safer to just append the two new items to the end of the existing order?